src/hb-glib.cc

```diff
@@ -246,7 +246,7 @@ hb_glib_unicode_compose (hb_unicode_funcs_t *ufuncs HB_UNUSED,
 {
 #if GLIB_CHECK_VERSION(2,29,12)
   return g_unichar_compose (a, b, ab);
-#endif
+#else
```

Comment:
See the comment below, this is intentionally left **not** `ifdef`’d out.

Reply:
Alternatively we need a bot this builds with an old GLib that hits the `else` branch, and remove the comment that is no longer true.

Reply:
Don't think we need to build, given that the code doesn't change.  But yeah, I prefer to compile the old code.  Then again, since this code won't ever change, I'm fine not compiling it.